source/agora/flash/Node.d

```diff
@@ -80,6 +81,9 @@ public abstract class FlashNode : FlashAPI
     /// any intermediaries.
     protected Hash[Hash] secrets;
 
+    /// Lightning Network topology
```

Comment:
I don't think we should use the Lightning network naming in our code. We want to differentiate our solution to the existing LN. Just Flash network is fine.